direct/io-s3/src/main/java/cz/o2/proxima/direct/s3/S3Client.java

```diff
@@ -135,9 +135,9 @@ private void validate() {
       Preconditions.checkArgument(!secretKey.isEmpty(), "secret-key must not be empty");
       @Nullable String base64SseKey = getOpt(cfg, "ssec-base64-key", Object::toString, null);
       boolean sslEnabled = getOpt(cfg, "ssl-enabled", Boolean::parseBoolean, false);
-      @Nullable URI endpoint = getOpt(cfg, "endpoint", URI::create, null);
+      @Nullable String endpoint = getOpt(cfg, "endpoint", Object::toString, null);
```

Comment:
Can we keep the URI and just add scheme when it is missing?

Reply:
It's used just in check for SSE-C encryption and just in case `ssl-enabled` is not specified. I don't see big value from this but ok, changed.

Reply:
It prevents putting in something, that cannot be interpreted as URI. Thanks. :+1: